fence/blueprints/login/fence_login.py

```diff
@@ -91,8 +91,20 @@ def get(self):
         tokens = flask.current_app.fence_client.fetch_access_token(
             redirect_uri, **flask.request.args.to_dict()
         )
+        # After Fence 5.0.0 "remove scopes from aud claim" changes,
+        # this validate_jwt is supposed to look like this:
+        #     id_token_claims = validate_jwt(
+        #         tokens["id_token"], scope="openid", purpose="id", attempt_refresh=True
```

Comment:
```suggestion
        #         tokens["id_token"], scope={"openid"}, purpose="id", attempt_refresh=True
```
should it be that? or i am focusing too much on past issues? 😆

Reply:
DANGIT no you're right ahahahahahaaha thank you boss